OneBlink.SDK/ApprovalsClient.cs

```diff
@@ -18,90 +19,70 @@ public ApprovalsClient(string accessKey, string secretKey, TenantName tenantName
             );
         }
 
-        public async Task<GetFormSubmissionAdministrationApprovalsResponse> GetFormSubmissionAdministrationApprovals(long formsAppId,
-        long limit,
-        long offset,
-        long? formId = null,
-        string externalId = null,
-        string submissionId = null,
-        string submittedAfterDateTime = null,
-        string submittedBeforeDateTime = null,
-        List<string> statuses = default(List<string>),
-        string updatedAfterDateTime = null,
-        string updatedBeforeDateTime = null)
+        public async Task<GetFormSubmissionAdministrationApprovalsResponse> GetFormSubmissionAdministrationApprovals(
+            long formsAppId,
+            long limit,
+            long offset,
+            long? formId = null,
+            string externalId = null,
+            Guid? submissionId = null,
+            string submittedAfterDateTime = null,
+            string submittedBeforeDateTime = null,
+            List<string> statuses = default(List<string>),
+            string updatedAfterDateTime = null,
+            string updatedBeforeDateTime = null,
+            List<string> lastUpdatedBy = null
+        )
         {
-            string queryString = nameof(limit) + "=" + limit.ToString() + "&" + nameof(offset) + "=" + offset.ToString();
+            NameValueCollection searchParams = System.Web.HttpUtility.ParseQueryString(string.Empty);

```

Comment:
```suggestion
            NameValueCollection searchParams = NameValueCollection();

```

Reply:
Changing to this suggestion causes the querystring to serialised to `/forms-apps/1640/approvals?System.Collections.Specialized.NameValueCollection`

Reply:
Interesting, looks like it relies on a bug https://stackoverflow.com/questions/3865975/namevaluecollection-to-url-query